lib/seattleflu/db/cli/command/etl/manifest.py

```diff
@@ -98,8 +102,15 @@ def etl_manifest(*, action: str):
                     mark_skipped(db, manifest_record.id)
                     continue
 
-                assert sample_identifier.set_name in expected_identifier_sets["samples"], \
-                    f"Sample identifier found in set «{sample_identifier.set_name}», not {expected_identifier_sets['samples']}"
+                if (manifest_record.document.get("sample_type") and
+                    manifest_record.document["sample_type"] == "rdt"):
+                    assert sample_identifier.set_name in expected_identifier_sets["rdt"], \
+                        (f"Sample identifier found in set «{sample_identifier.set_name}», " +
+                        f"not {expected_identifier_sets['rdt']}")
+                else:
+                    assert sample_identifier.set_name in expected_identifier_sets["samples"], \
+                        (f"Sample identifier found in set «{sample_identifier.set_name}», " +
+                        f"not {expected_identifier_sets['samples']}")
 
```

Comment:
I think a few formatting changes snuck into this commit (lines 105-113 and 141-142 in `etl/manifest`).